PWGHF/HFL/Tasks/taskSingleMuonMult.cxx

```diff
@@ -0,0 +1,296 @@
+        auto dpt(muonType3.pt() - pt);
+        if (muTrackType == 0) {
+          registry.fill(HIST("hMuDeltaPtBeforeAccCuts"), cent, nCh, pt, eta, theta, rAbs, dcaXY, pDca, chi2, dpt);
+        }
+      }
+
+      // Apply various standard muon acceptance cuts
+      // eta cuts
+      if ((eta >= etaMax) || (eta < etaMin)) {
+        continue;
+      }
+      registry.fill(HIST("hMuTrkSel"), 2);
+      registry.fill(HIST("hMuAfterEtaCuts"), cent, nCh, pt, eta, theta, rAbs, dcaXY, pDca, chi2, muTrackType);
+
+      // Rabs cuts
+      if ((rAbs < rAbsMin) || (rAbs >= rAbsMax)) {
+        continue;
+      }
+      registry.fill(HIST("hMuTrkSel"), 3);
+      registry.fill(HIST("hMuAfterRabsCuts"), cent, nCh, pt, eta, theta, rAbs, dcaXY, pDca, chi2, muTrackType);
+
+      if ((rAbs < rAbsMid) && (pDca >= pDcaMin)) {
+        continue;
+      }
+      if ((rAbs >= rAbsMid) && (pDca >= pDcaMax)) {
+        continue;
+      }
+      registry.fill(HIST("hMuTrkSel"), 4);
+      registry.fill(HIST("hMuAfterPdcaCuts"), cent, nCh, pt, eta, theta, rAbs, dcaXY, pDca, chi2, muTrackType);
+
+      //  MCH-MFT matching chi2
+      if ((muon.chi2() >= 1e6) || (muon.chi2() < 0)) {
+        continue;
+      }
+      registry.fill(HIST("hMuTrkSel"), 5);
+
+      // histograms after acceptance cuts
+      registry.fill(HIST("hMuAfterAccCuts"), cent, nCh, pt, eta, theta, rAbs, dcaXY, pDca, chi2, muTrackType);
+      registry.fill(HIST("h3DCAAfterAccCuts"), muon.fwdDcaX(), muon.fwdDcaY(), muTrackType);
+      nMu++;
+      if (muTrackType == 0) {
+        nMu0++;
+      }
+      if (muTrackType == 1) {
+        nMu1++;
+      }
+      if (muTrackType == 2) {
+        nMu2++;
+      }
+      if (muTrackType == 3) {
+        nMu3++;
+      }
+      if (muTrackType == 4) {
+        nMu4++;
+      }
+
+      if (muon.has_matchMCHTrack()) {
+        auto muonType3 = muon.template matchMCHTrack_as<MyMuons>();
+        auto dpt(muonType3.pt() - pt);
+
+        if (muTrackType == 0) {
+          registry.fill(HIST("hMuDeltaPtAfterAccCuts"), cent, nCh, pt, eta, theta, rAbs, dcaXY, pDca, chi2, dpt);
+        }
+      }
+    }
+
+    registry.fill(HIST("h3MultNchNmu"), cent, nCh, nMu);
+    if (nMu0 > 0)
+      registry.fill(HIST("h3MultNchNmu_TrackType0"), cent, nCh, nMu0);
+    if (nMu1 > 0)
+      registry.fill(HIST("h3MultNchNmu_TrackType1"), cent, nCh, nMu1);
+    if (nMu2 > 0)
+      registry.fill(HIST("h3MultNchNmu_TrackType2"), cent, nCh, nMu2);
+    if (nMu3 > 0)
+      registry.fill(HIST("h3MultNchNmu_TrackType3"), cent, nCh, nMu3);
+    if (nMu4 > 0)
+      registry.fill(HIST("h3MultNchNmu_TrackType4"), cent, nCh, nMu4);
+    chTracks.clear();
+  }
+
+  void processMuon(MyCollisions::iterator const& collision,
+                   MyTracks const& tracks,
+                   MyMuons const& muons)
+  {
+    runMuonSel(collision, tracks, muons);
+  }
+  PROCESS_SWITCH(HfTaskSingleMuonMult, processMuon, "run muon selection with real data", true);
```

Comment:
You don't need a process function switch for a single process function. Just call it `process`.

Reply:
Yes